browse/src/main/kotlin/com/alfresco/content/browse/processes/details/ProcessDetailViewModel.kt

```diff
@@ -243,12 +243,14 @@ class ProcessDetailViewModel(
                     is Success -> {
                         val response = it()
 
-                        repository.saveSourceName(response.listAccounts.first())
-                        val sourceName = response.listAccounts.first().sourceName
-                        if (!isExecuted) {
-                            isExecuted = true
-                            state.parent?.defaultEntries?.map { entry ->
-                                linkContentToProcess(entry, sourceName)
+                        if(response.listAccounts.isNotEmpty()){
```

Comment:
If there is any data in the list then it will fall in the Success block otherwise handled by the Fail case. So it wouldn't crash here.